crates/router/src/core/payouts.rs

```diff
@@ -327,6 +334,73 @@ pub async fn payouts_create_core(
     )
     .await?;
 
+    if payout_data.payouts.confirm == Some(true) {
+        payouts_core(
+            &state,
+            &merchant_account,
+            &key_store,
+            &mut payout_data,
+            req.routing.clone(),
+            req.connector.clone(),
+        )
+        .await?
+    };
+
+    response_handler(&merchant_account, &payout_data).await
+}
+
+#[instrument(skip_all)]
+pub async fn payouts_confirm_core(
+    state: SessionState,
+    merchant_account: domain::MerchantAccount,
+    key_store: domain::MerchantKeyStore,
+    req: payouts::PayoutCreateRequest,
+) -> RouterResponse<payouts::PayoutCreateResponse> {
+    let _payout_id = req.payout_id.clone().get_required_value("payout_id")?;
```

Comment:
Why are we asking for `payout_id` in the request but then dropping the value later?

Reply:
Will remove